yaml.qmd

```diff
@@ -0,0 +1,874 @@
+For example, the `name` key nested under `author` is indented two spaces, whereas the `name` key nested under `affiliation`, which itself is nested under `author`, is indented four spaces:
+
+``` {.yaml filename="document.qmd"}
+---
+title: My document
+author:
+  name: Soraya Drake
+  affiliation: 
+    name: Everpine University    
+    url: http://everpine.edu
+---
+```
+
+The names you can use in Quarto, along with the types of values they take, are not specified by the YAML language, but by Quarto's own schema.
+
+### YAML Sequences
+
+A YAML sequence is a way to provide a key a set of ordered values.
+There are two syntaxes to specify a YAML sequence.
+When the values in the sequence are simple, separate them with commas inside square brackets:
+
+``` {.yaml filename="document.qmd"}
+---
+theme: [cosmo, styles.scss]
+---
+```
+
+The alternate syntax puts each value on their own line, and adds a dash prefix:
+
+``` {.yaml filename="document.qmd"}
+---
+theme: 
+  - cosmo
+  - styles.scss
+---
+```
+
+This alternate syntax can also handle situations where the values are further mappings.
+For example, here the value of `author` is a sequence of length two, where each element is a mapping of `name` and `affiliation`:
+
+``` {.yaml filename="document.qmd"}
+---
+title: My document
+author:
+  - name: Soraya Drake
+    affiliation: Everpine University    
+  - name: Zahir Kamarov
+    affiliation: Peregrine Heights Institute
+---
+```
+
+### YAML Scalars
+
+YAML uses the term *scalar* to talk about the simplest types of value: strings, numbers, and booleans.
+You saw an example of each earlier:
+
+``` {.yaml filename="document.qmd"}
+---
+title: My Document  # A string
+toc: true           # A boolean, `true`/`false`
+toc-depth: 2        # A number
+---
+```
+
+The only values recognized as boolean are `true` and `false`, numbers can be single integers (`2`), decimal (`2.1`) or scientific (`2.1e+2`), and anything else is treated as a string.
+
+### Quotes
+
+Strings don't require quotes, but you may need them if your string contains characters that have special meaning in YAML (e.g. `#`, `:`, etc.), or is something that would otherwise be interpreted as a number or boolean:
+
+``` {.yaml filename="document.qmd"}
+---
+title: "Quotes: and things that need them"  
+author: "#quarto"
+subtitle: "100"
+---
+```
+
+In YAML double quoted strings (`"`) allow escape characters in them, whereas single quoted strings (`'`) are interpreted literally:
+
+``` {.yaml filename="document.qmd"}
+---
+title: "Quotes: \U0001F4E8"
+subtitle: 'Quotes: \U0001F4E8'
+---
+```
+
+Q: where does the `\U` go in `subtitle` go?
+YAML might be interpreting literally but Quarto/Pandoc might escape somewhere?
+
+### Long strings
+
+For strings that are longer than you'd comfortably put on a single line, or need explicit line breaks, you can use what YAML calls block scalars.
```

Comment:
What should we use to call out terms? Bolding? e.g., around **block scalars**.

Relevant but not just about this question -- are we doing an index?